pyproject.toml

```diff
@@ -40,7 +41,8 @@ dev = [
     "sphinx",
     "myst-parser",
     "sphinx-rtd-theme",
-    "git+https://github.com/executablebooks/sphinx-proof@d37495f",
+    # This results in an error when running setup.py:
```

Comment:
That I will have to look into. Last time I checked, we cannot build the docs wihtout this specific version, which contains / contained unreleased features. But also, the development dependencies should not be involved in the package. I will have to look into that.

Reply:
So, I checked. There is no new release of this package yet, so we still depend on it. Please revert this change, and I will take a look into the packaging.